include/freerdp/update.h

```diff
@@ -231,7 +232,8 @@ struct rdp_update
 	 * fills BITMAP_DATA struct members: flags, cbCompMainBodySize and cbCompFirstRowSize.
 	 */
 	BOOL autoCalculateBitmapData; /* 71 */
-	UINT32 paddingE[80 - 72];     /* 72 */
+	pServerCutText ServerCutText; /* 72 */
```

Comment:
why is this necessary in the public API?

Reply:
See the previous answer.